seam/paginator.py

```diff
@@ -0,0 +1,102 @@
+from typing import Callable, Dict, Any, Tuple, Generator, List
+
+
+class Pagination:
+    def __init__(
+        self,
+        has_next_page: bool,
+        next_page_cursor: str | None,
+        next_page_url: str | None,
+    ):
+        self.has_next_page = has_next_page
+        self.next_page_cursor = next_page_cursor
+        self.next_page_url = next_page_url
```

Comment:
Move below `Paginator` class

Reply:
I tried that initially but got an error in the editor
![image](https://github.com/user-attachments/assets/594efdb2-8bf2-4178-879c-3f4cc52c9074)

Reply:
Right, python doesn't hoist. This is more like a resource class then 🤔 Ok to leave here but it could live in `pagination.py` on its own.